include/Foundation/NSLayoutAnchor.h

```diff
@@ -0,0 +1,23 @@
+/*
+ This file is part of Darling.
+
+ Copyright (C) 2025 Darling Developers
+
+ Darling is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Darling is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Darling.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#import <Foundation/Foundation.h>
+
+@interface NSLayoutDimension : NSObject
```

Comment:
[`NSLayoutDimension` extends from `NSLayoutAnchor`](https://developer.apple.com/documentation/appkit/nslayoutdimension?language=objc). Make sure to create a `NSLayoutAnchor` stub as well!
```obj-c
@interface NSLayoutDimension : NSLayoutAnchor
```